### ISSUE TITLE
Improve Discovery and Peering documentation

### ISSUE THREAD

**Issue by amedranogil** [universAAL/middleware#9]
I have been reading : https://github.com/universAAL/middleware/wiki/Discovery-and-Peering
And it feels there are many unanswered questions like:
- what are the different possibilities for the Space Profile in the AALSpace descriptor?
- what are the default values for each of the properties of the AALSpace descriptor, if not provided? is there a default AALSpace?
- what exactly are the domain Attributes used for?
- how exactly are the security cards configured and used?
- how are the AALSpace operations executed? 
  * I assume these are descriptions for the AALSpace manager, but there is no mention of it. 
  * Also from a deployer perspective, these operations should be done through the OSGi console commands, but there is no specification nor mention to this. It would be nice to have a wiki for these commands and link to this wiki here, or have some sample commands for each operation. maybe these descriptions should be extended in https://github.com/universAAL/platform/wiki/RD-Middleware
  * I also assume the AALSpace description are stored in the *.space files within the config_dir; which also brings the possibility to configure the spaces offline (without/before running the middleware). But there is no guide to do this (the specs of the xml schema are a bit opaque and the namesapce http://universaal.org/aalspace-channel/v1.0.0 seems to be offline)
- the wiki states that joining an AALSpace can be delegated to a trusted service ...  how? this will be very interesting to implement in the very near future.
- it is not specified if each node can join more than one space at a time, I assume not. Yet it would be interesting for roaming nodes to join specially public spaces, simultaneously to their own. But of course this poses loads of privacy and security issues (discussion which should be started somewhere else) 
- how are the encrypted messages configured? how are the keys configured, shared, managed, etc...? the wiki states the key is stored in the config_dir (according to recommended property configuration) but it does not detail exactly which file needs to be added, nor how to create a new file (contents?)... not really an answer but it seems to be here: https://github.com/universAAL/platform/wiki/RD-Middleware#apache-karaf-configuration-files

**Comment by amedranogil** [comment]
is it possible that a coordinator requests other peers to join?
Use Case: the coordinator is a node with UI, there is a manager that lists available peers, where the user can select the remote nodes to join the space. With in these nodes there may be new devices, without UI, which the user wishes they join the space.

**Comment by michelegirolami** [comment]
@amedranogil  no, your use case is not possibile. The "join" message is issued only by a node (wither PEER or COORDINATOR) willing to join to the AAL space found. Currently, no "INVITE" message is implemented.
The COORDINATOR can accept or deny the join request.

**Comment by amedranogil** [comment]
Is it possible to configure more than one AALSPace per node? 
Use Case: I have an android phone on it's self is an AALSpace, but when it connects to a certain network (wifi at the service provider premices) it can access the AALSpace set up in that network.
